model/src/main/scala/za/co/absa/atum/model/Checkpoint.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.atum.model
+
+import java.time.ZonedDateTime
+
+case class Checkpoint(
+    name: String,
+    partitioning: Partitioning,
+    processStartTime: ZonedDateTime,
+    processEndTime: ZonedDateTime,
+    measurements: Seq[Measurement]
```

Comment:
```suggestion
    measurements: Seq[Measurement],
    author: String
```
We need to add the support for "authoring" into `AtumAgent` and `AtumContext` methods.

Reply:
okay